examples/usgs-stac-convert/mss/LM01_L1GS_001010_19720908_02_T2.json

```diff
@@ -0,0 +1,318 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "LM01_L1GS_001010_19720908_02_T2",
+  "properties": {
+    "datetime": "1972-09-08T13:43:34.091000Z",
+    "eo:cloud_cover": 43.0,
+    "view:sun_azimuth": 172.41815593,
+    "view:sun_elevation": 24.87312023,
+    "platform": "landsat-1",
+    "instruments": [
+      "mss"
+    ],
+    "view:off_nadir": 0,
+    "landsat:cloud_cover_land": 46.0,
+    "landsat:wrs_type": "1",
+    "landsat:wrs_path": "001",
+    "landsat:wrs_row": "010",
+    "landsat:scene_id": "LM10010101972252XXX01",
+    "landsat:collection_category": "T2",
+    "landsat:collection_number": "02",
+    "landsat:correction": "L1GS",
+    "proj:epsg": 32625,
+    "description": "Landsat Collection 2 Level-1 Product",
+    "created": "2022-02-19T16:04:27.605831Z"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          -34.342777359690906,
+          71.6736886240419
+        ],
+        [
+          -36.70080544861747,
+          70.18448226408043
+        ],
+        [
+          -32.42683403196701,
+          69.42483853001222
+        ],
+        [
+          -29.841815210171347,
+          70.85738280018273
+        ],
+        [
+          -34.342777359690906,
+          71.6736886240419
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "alternate",
+      "href": "https://landsatlook.usgs.gov/stac-browser/collection02/level-1/standard/mss/1972/001/010/LM01_L1GS_001010_19720908_20200909_02_T2",
+      "type": "text/html",
+      "title": "USGS stac-browser page"
+    },
+    {
+      "rel": "self",
+      "href": "/Users/pjh/dev/landsat/examples/usgs-stac-convert/mss/LM01_L1GS_001010_19720908_02_T2.json",
+      "type": "application/json"
+    }
+  ],
+  "assets": {
+    "thumbnail": {
+      "href": "tests/data-files/usgs-stac/mss/LM01_L1GS_001010_19720908_20200909_02_T2_thumb_small.jpeg",
```

Comment:
Try using `make_asset_hrefs_relative()` to get valid relative paths, e.g. https://github.com/stactools-packages/modis/blob/main/scripts/create_expected.py#L35.

Reply:
Hmm. Setting `item.make_asset_hrefs_relative()` does not change the output. Are you thinking that it should show something along the lines of `../../tests/data-files/...`? If I use `item.make_asset_hrefs_absolute()`, then the path is full, e.g., `/Users/pjh/dev/...`.